### PR TITLE
feat(design): update daffodil's brand palette to turqoise and create green palette to use for success UI

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Missing a palette for 'success' state usage.

Fixes: N/A

## What is the new behavior?
Update Daffodil's tertiary color palette name to `daff-turquoise` and create a `daff-green` palette that can be used to style states relevant to success / completion.

## Does this PR introduce a breaking change?
```
[x] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: libs/design/src/scss/theming/_variables.scss [hunk 1]
```diff
@@ -87,9 +87,22 @@ $daff-bronze: (
...
+	70: #0e6726,
+	80: #0f4c1e,
+	90: #0c3515,
+	100: #07230d,
+);
+
 $daff-gray: (
 	0: #ffffff,
-	10: #f7f7f7,
+	10: #fafafa,
```
Comment (damienwebdev) [inline libs/design/src/scss/theming/_variables.scss:105]: Different PR please with the motivation for the change.

File: libs/design/src/scss/theming/_variables.scss [hunk 2]
```diff
@@ -87,9 +87,22 @@ $daff-bronze: (
 	100: #1a0f00,
 );
 
+$daff-green: (
```
Comment (damienwebdev) [inline libs/design/src/scss/theming/_variables.scss:90]: Please modify the commit to include the `BREAKING CHANGE:` explaining why you broke the API.

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: libs/design/src/scss/theming/_variables.scss (hunk 2 above)
Comment (xelaint) [reply libs/design/src/scss/theming/_variables.scss:90]: updated

File: libs/design/src/scss/theming/_variables.scss (hunk 1 above)
Comment (xelaint) [reply libs/design/src/scss/theming/_variables.scss]: removed

**Review by damienwebdev — APPROVED** [review]
lgtm

**Pull request merged**